obelisk_ws/src/cpp/obelisk_viz_cpp/src/default_robot_viz.cpp

```diff
@@ -0,0 +1,7 @@
+#include "obelisk_ros_utils.h"
+#include "obelisk_viz_robot_default.h"
+
+int main(int argc, char* argv[]) {
+    obelisk::utils::SpinObelisk<obelisk::viz::ObeliskVizRobotDefault<obelisk_estimator_msgs::msg::EstimatedState>,
+                                rclcpp::executors::MultiThreadedExecutor>(argc, argv, "robot_viz");
```

Comment:
does this need to be a multithreaded executor?

Reply:
It can be - and I actually have programmed in protections for the multi threaded case, but no, it does not need to be.